### DISCUSSION TITLE
Is it time for 1.0?

### DISCUSSION THREAD

**Discussion by pwaller** [pwaller/pyfiglet#98] [Announcements]
pyfiglet is moving very slowly at this point, and I don't imagine breaking any API in the future. Maybe it's time for 1.0. Comments and suggestions welcome!

**Comment by DTheIcyDragon** [comment]
I'd like an 1.0, it is a (as far as I can tell) very well made package wich I love using for ascii art in terminal based applications.

**Comment by Siddhesh-Agarwal** [comment]
Honestly, Pyfiglet seems stable enough for a 1.0.0 release.

**Comment by pwaller** [comment]
Some time passes, and now there is a v1.0.0rc1 tag. Please test and report back.

**Comment by pwaller** [comment]
v1.0.0 is released, thanks everyone for your help, in particular @peterbrittain and @stefanor.
